eland/tests/series/test_na_pytest.py

```diff
@@ -0,0 +1,34 @@
+# Licensed to Elasticsearch B.V under one or more agreements.
+# Elasticsearch B.V licenses this file to you under the Apache 2.0 License.
+# See the LICENSE file in the project root for more information
+
+from eland import eland_to_pandas
+from eland.tests.common import TestData
+from eland.tests.common import assert_pandas_eland_frame_equal
+
+
+class TestSeriesNA(TestData):
+    columns = [
+        "currency",
+        "customer_full_name",
+        "geoip.country_iso_code",
+        "geoip.region_name",
+    ]
+
+    def test_isna(self):
+        ed_ecommerce = self.ed_ecommerce()
+        pd_ecommerce = eland_to_pandas(ed_ecommerce)
+
+        for column in self.columns:
+            isna_ed_ecommerce = ed_ecommerce[~ed_ecommerce[column].isna()]
```

Comment:
Can we also test `.isna()` where it's not inverted

Reply:
Done